tests/DependencyInjection/NijensOpenapiExtensionTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the OpenapiBundle package.
+ *
+ * (c) Niels Nijens <nijens.niels@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nijens\OpenapiBundle\Tests\DependencyInjection;
+
+use JsonSchema\Validator;
+use League\JsonReference\Dereferencer;
+use League\JsonReference\ReferenceSerializer\InlineReferenceSerializer;
+use Nijens\OpenapiBundle\Controller\CatchAllController;
+use Nijens\OpenapiBundle\DependencyInjection\NijensOpenapiExtension;
+use Nijens\OpenapiBundle\EventListener\JsonRequestBodyValidationSubscriber;
+use Nijens\OpenapiBundle\EventListener\JsonResponseExceptionSubscriber;
+use Nijens\OpenapiBundle\Json\SchemaLoader;
+use Nijens\OpenapiBundle\Routing\RouteLoader;
+use Nijens\OpenapiBundle\Service\JsonResponseBuilderException;
+use PHPUnit\Framework\TestCase;
+use Seld\JsonLint\JsonParser;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * NijensOpenapiExtensionTest.
+ *
+ * @author David Cochrum <dcochrum@sonnysdirect.com>
+ */
+class NijensOpenapiExtensionTest extends TestCase
```

Comment:
I understand why you would want 100% code coverage. Though, I'd rather not have it added in this pull request. Also, I'd rather see something like [matthiasnoback/symfony-dependency-injection-test](https://packagist.org/packages/matthiasnoback/symfony-dependency-injection-test) implemented in a test case.

Reply:
Apparently with the classes and tests I added, the coverage dropped 0.01% and coveralls was blocking acceptance

Reply:
Ah... Let's just leave it here then.

Reply:
Leave what where? I don't mind updating the test to use that lib

Reply:
I meant you can keep the test case as-is for now. But if you'd like (and have time to spare of course) it would be awesome when you added that library.